_posts/2021-08-11-Day-3-Post.md

```diff
@@ -0,0 +1,112 @@
+
+## Implementing the Metropolis Criteron
+
+To accomplish this, we are going to implement acceptance critera, specifically the *Metropolis-Hasting* algorith, to determine if we will accept a change in the position of a particle in a system considering the changes in the system energy that the positional shift imparts. The *Metropolis-Hasting* criteron we built into our model is as follows:
+
+$$ P_acc *P<sub>acc</sub>(m → n)* = min[1,*e*<sup>-$\Delta$U/*T*</sup>] $$
+
+{% highlight python %}
+def accept_or_reject(delta_U, beta):
+    if delta_U <= 0.0:
+        accept = True
+    else:
+        #Generate a random number on (0,1)
+        random_number = random.random()
+        p_acc = math.exp(-beta*delta_U)
+        
+        if random_number < p_acc:
+            accept = True
+        else:
+            accept = False
+    return accept
+{% endhighlight %}
+
+As we have established our system, if a configuration is energetically more favorable, it is always accepted. If a configuration is more energetically "uphill", a number is generated randomly on (0,1) and compared to the expression *e*<sup>-$\Delta$U/*T*</sup>. If the generated number is less than expression, the criteron is not met, and the change is rejected. By this model, all configurations with lower energy or always accepted, configuration with increases in energy are accept a fraction of the time, and configurations that have great increases in energy are possible, but very unlikely.
+
+We continuted to investigate on the relation between the expression *e*<sup>-$\Delta$U/*T*</sup> and the probability density. Upon plotting the probility density against $\Delta$U from (-2,2) for different values of T the following graph was generated.
+
+<center>
+
+<img src = '{{ "/images/MCTemp.PNG" | relative_url }}'>  
+
+***Figure 1**: Probability density visualized against various $\Delta$U for three different temperatures*
+</center>
+
+Upon discussing our modelings, we noted that for the domain $\Delta$U > 0, if the temperature of the system is higher, the propability density is also higher. This means that at higher temperatures, systems are more likely to enter states of higher energy. We also noted that for all $\Delta$U < 0, the generated *P<sub>acc</sub>* was above 1.0 for all temperatures. However, because this value was only compared to a generated number from (0,1), the shift is effectively accepted 100% of the time. This lines up with our assumption that all configurations of lower energy are accepted.
```

Comment:
```suggestion
Upon discussing our models, we noted that for the domain $\Delta$U > 0, if the temperature of the system is higher, the probability density is also higher. This means that at higher temperatures, systems are more likely to enter states of higher energy. We also noted that for all $\Delta$U < 0, the generated *P<sub>acc</sub>* was above 1.0 for all temperatures. However, because this value was only compared to a generated number from (0,1), the shift is effectively accepted 100% of the time. This lines up with our assumption that all configurations of lower energy are accepted.
```